### ISSUE TITLE
Planner 3 beta 2 doesn't work at all

### ISSUE THREAD

**Issue by ChibiYuzu** [alainm23/planify#929]
**Describe the bug**
Yesterday, I discovered that Planner had been upgraded to a version 3 beta 2 and there were lots of things missing, including my past data (which seems to have been wiped out) and the possibility to import backups made with a previous version (2.7), while clicking on some button would make it crash altogether. I uninstalled it and tried to re-install Planner 2.7, but to no avail, despite your instructions. And now, when I re-installed it from FlatHub, it simply crashes upon launching. 

**Expected behavior**
I wish that:
1. the Flatpack version would launch properly
2. it would be possible to import backups made with previous versions
3. all the buttons in it would work
4. one could create section in projects again
5. one would be able to add sub-tasks to tasks
6. one would be able to have the task repeated
7. one could add categories to the tasks
8. one could add description notes to projects

P.S.: Other people have been complaining about issues 2-6, but on a per-issue basis. I have all of them together!

**Desktop (please complete the following information):**
 - Ubuntu 22.04 (Elementary OS 7 early access), GNOME 42.2
 - Panner version 3 beta 2

**Comment by ChibiYuzu** [comment]
Hello again!

I have tried re-installing it using command lines as shown on the Flatpak page for your application. Here is what happens:

****
Mycomputer:~$ flatpak install flathub com.github.alainm23.planner
Looking for matches…

com.github.alainm23.planner permissions:
 ipc       network       fallback-x11          wayland
x11       dri           file access [1]       dbus access [2]

[1] home
 [2] org.gnome.evolution.dataserver.Calendar8,
     org.gnome.evolution.dataserver.Sources5

  ID                                 Branch Op Remote  Download
  ID                                 Branch Op Remote  Download
 1. [✓] com.github.alainm23.planner.Locale stable i  flathub 12.1 kB / 3.4 MB
 2. [✓] com.github.alainm23.planner        stable i  flathub 27.7 MB / 28.3 MB

Installation complete.
Mycomputer:~$ flatpak run com.github.alainm23.planner
Gtk-Message: 11:29:41.475: Failed to load module "xapp-gtk3-module"
Gtk-Message: 11:29:41.475: Failed to load module "appmenu-gtk-module"
Gtk-Message: 11:29:41.605: Failed to load module "canberra-gtk-module"
Gtk-Message: 11:29:41.607: Failed to load module "canberra-gtk-module"

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.575: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.579: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.583: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.587: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.594: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.598: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.601: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.612: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.615: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.619: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.627: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.631: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.636: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.645: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.649: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.652: gee_map_set: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.652: services_database_get_items_by_scheduled: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.652: gee_abstract_collection_get_size: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.721: gee_abstract_map_get_values: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.721: gee_iterable_iterator: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.721: gee_iterator_next: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.722: gee_abstract_map_get_values: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.722: gee_iterable_iterator: assertion 'self != NULL' failed

** (com.github.alainm23.planner:2): CRITICAL **: 11:29:42.722: gee_iterator_next: assertion 'self != NULL' failed

(com.github.alainm23.planner:2): GLib-GObject-WARNING **: 11:29:42.722: invalid (NULL) pointer instance

(com.github.alainm23.planner:2): GLib-GObject-CRITICAL **: 11:29:42.722: g_signal_connect_object: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

(com.github.alainm23.planner:2): GLib-GObject-WARNING **: 11:29:42.722: invalid (NULL) pointer instance

(com.github.alainm23.planner:2): GLib-GObject-CRITICAL **: 11:29:42.722: g_signal_connect_object: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

(com.github.alainm23.planner:2): GLib-GObject-WARNING **: 11:29:42.722: invalid (NULL) pointer instance

(com.github.alainm23.planner:2): GLib-GObject-CRITICAL **: 11:29:42.722: g_signal_connect_object: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed
**
ERROR:../src/Services/CalDAV/Util.vala:134:cal_dav_util_ical_to_date_time_local: assertion failed: (!date.is_null_time ())
Bail out! ERROR:../src/Services/CalDAV/Util.vala:134:cal_dav_util_ical_to_date_time_local: assertion failed: (!date.is_null_time ())
mycomputer:~$ 

****
I have tried making sure that I have the following modules, which failed to load, on my computer and even re-installed them, thinking they might be dysfunctioning, but to no avail:
xapp-gtk3-module
appmenu-gtk-module
canberra-gtk-module
canberra-gtk-module

So basically, it seems like it doesn't work at all anymore on my computer. I can't even install it.  My system specifications haven't changed since the previous message above.

**Comment by zhrexl** [comment]
I can confirm the flatpak version is crashing on launch. How can I give more infos about it?

**Comment by ChibiYuzu** [comment]
It seems now that Planner is launching properly. At least, now that I have replaced Ubuntu 22.04 with Linux Mint 21. However, there are a lot of things that have disappeared, so that makes it much less useful to me than it was before:
There are no more:
- tags for tasks
- sub-tasks
- repetition of tasks
- sections
And I still can't import data from previous versions of Planner, forcing me to re-write all my projects, tasks, etc. 
So, for the moment, I don't think I'll continue using it, because it's just making things more tedious for me.

**Comment by 0chroma** [comment]
this seems to be a duplicate of or related to #911
